.github/workflows/deploy-enterprise-prod.yml

```
@@ -0,0 +1,88 @@
+name: Deploy Enterprise prod to AWS
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - environment: hyp3-enterprise
+            domain: hyp3-enterprise.asf.alaska.edu
+            template_bucket: cf-templates-qgq5ahzym464-us-west-2
+            image_tag: latest
+            product_lifetime_in_days: 30
+            default_credits_per_user: 0
+            default_application_status: APPROVED
+            cost_profile: DEFAULT
```

Maybe let's use EDC costs for now, since they're closer to what the final costs will be:
```suggestion
            cost_profile: EDC # TODO: update before going public
```